src/Graphics/UI.h

```diff
@@ -75,7 +75,7 @@ class UI
     int _y = 0;
     int _xOffset = 0;
     int _yOffset = 0;
-    Texture* _texture = 0;
+    mutable Texture* _texture = 0;
```

Comment:
I made it mutable to have a const texture() getter.
It is used in width() and height() getters and that's why it should be const.
I understand that this solution isn't perfect (